pkg/cloudprovider/controller/nodecontroller.go

```diff
@@ -104,6 +107,14 @@ func NewNodeController(
 	nodeMonitorGracePeriod time.Duration,
 	nodeStartupGracePeriod time.Duration,
 	nodeMonitorPeriod time.Duration) *NodeController {
+	eventBroadcaster := record.NewBroadcaster()
+	recorder := eventBroadcaster.NewRecorder(api.EventSource{Component: "controllermanager"})
+	if kubeClient != nil {
+		glog.Infof("Sending events to api server.")
+		eventBroadcaster.StartRecordingToSink(kubeClient.Events(""))
```

Comment:
I thought we agreed that node events would go in a namespace associated with the cluster-id . 

@thockin @bgrant0607 am I mis-remembering the outcome of that discussion?